cli/ballerina-cli/src/main/resources/cli-help/ballerina-semver.help

```diff
@@ -0,0 +1,34 @@
+NAME
```

Comment:
Should we update general help with subcommand (`bal help`)

Reply:
Yes that would be better. +1

Reply:
Added with [4df4825](https://github.com/ballerina-platform/ballerina-lang/pull/36259/commits/4df4825bbf9e0b21b9c38d014188450b74fb59e2)